### PR TITLE
docs(@babel/parser): Document createParenthesizedExpressions option.

### PR DESCRIPTION
**Pull request by toshi-toma**
Documents the changes from babel/babel#8025 implemented by @arv

### CODE REVIEW

File: docs/parser.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ mind. When in doubt, use `.parse()`.
   to set this option to `true` to prevent the parser from prematurely
   complaining about undeclared exports that will be added later.
 
+- **createParenthesizedExpressions**: By default, the parser sets extra.parenthesized on the expression nodes. When this option is set to `true`, create ParenthesizedExpression AST nodes.
```
Comment (nicolo-ribaudo) [inline docs/parser.md]: Nit: Wrap `ParenthesizedExpression` with `

Comment (toshi-toma) [reply docs/parser.md]: fixes a7750c7

File: docs/parser.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ mind. When in doubt, use `.parse()`.
   to set this option to `true` to prevent the parser from prematurely
   complaining about undeclared exports that will be added later.
 
+- **createParenthesizedExpressions**: By default, the parser sets extra.parenthesized on the expression nodes. When this option is set to `true`, create `ParenthesizedExpression` AST nodes.
```
Comment (arv) [inline docs/parser.md]: Maybe

```
create `ParenthesizedExpression` AST nodes instead.
```

File: website/versioned_docs/version-7.4.0/parser.md [hunk 3]
```diff
@@ -47,6 +47,8 @@ mind. When in doubt, use `.parse()`.
   outside of class and object methods. Set this to `true` to accept such
   code.
 
+- **createParenthesizedExpressions**: By default, the parser sets extra.parenthesized on the expression nodes. When this option is set to `true`, create `ParenthesizedExpression`  AST nodes.
```
Comment (arv) [inline website/versioned_docs/version-7.4.0/parser.md]: extra space before `AST nodes`

Comment (existentialism) [reply website/versioned_docs/version-7.4.0/parser.md]: 👍

**Comment by existentialism** [comment]
@toshi-toma thanks!

File: website/versioned_docs/version-7.4.0/parser.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ mind. When in doubt, use `.parse()`.
   outside of class and object methods. Set this to `true` to accept such
   code.
 
-- **createParenthesizedExpressions**: By default, the parser sets extra.parenthesized on the expression nodes. When this option is set to `true`, create `ParenthesizedExpression`  AST nodes.
+- **createParenthesizedExpressions**: By default, the parser sets `extra.parenthesized` on the expression nodes. When this option is set to `true`, `ParenthesizedExpression` AST nodes are created instead.
```
Comment (arv) [inline website/versioned_docs/version-7.4.0/parser.md]: This snippet is repeated in 4 places. Can you update them all?

**Comment by arv** [comment]
LGTM

Thanks.

**Pull request merged**